specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -2443,6 +2620,262 @@
+        },
+        "temperature": {
+          "$ref": "#/definitions/WeatherUnitMaxMinAvg",
+          "description": "Temperature values."
+        },
+        "degreeDaySummary": {
+          "$ref": "#/definitions/DegreeDaySummary",
+          "description": "Summary of heating or cooling degree day information"
+        },
+        "precipitation": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "The amount of precipitation (liquid equivalent) that has fallen."
+        }
+      }
+    },
+    "HistoricalSummaryResponse": {
+      "type": "object",
+      "properties": {
+        "historicalActuals": {
+          "$ref": "#/definitions/MonthlyHistoricalActuals",
+          "description": "Historical actuals for the requested month."
+        },
+        "historicalRecords": {
+          "$ref": "#/definitions/MonthlyHistoricalRecords",
+          "description": "Historical records for the requested month."
+        },
+        "historicalNormals": {
+          "$ref": "#/definitions/MonthlyHistoricalNormals",
+          "description": "Historical normals for the requested month."
+        }
+      }
+    },
+    "MonthlyHistoricalActuals": {
+      "type": "object",
+      "properties": {
+        "temperature": {
+          "$ref": "#/definitions/WeatherUnitMaxMinAvg",
+          "description": "Temperature value."
+        },
+        "degreeDaySummary": {
+          "$ref": "#/definitions/DegreeDaySummary",
+          "description": "Summary of Heating or Cooling Degree Day information."
+        },
+        "precipitation": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "The amount of precipitation (liquid equivalent) that has fallen."
+        },
+        "snow": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "Snow."
+        },
+        "snowDepth": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "Snow depth."
+        }
+      }
+    },
+    "MonthlyHistoricalRecords": {
+      "type": "object",
+      "properties": {
+        "temperature": {
+          "$ref": "#/definitions/WeatherUnitYearMaxMinAvg",
+          "description": "Temperature value."
+        },
+        "precipitation": {
+          "$ref": "#/definitions/WeatherUnitYearMax",
+          "description": "Maximum amount of precipitation (liquid equivalent) that has fallen."
+        },
+        "snow": {
+          "$ref": "#/definitions/WeatherUnitYearMax",
+          "description": "Maximum snow."
+        }
+      }
+    },
+    "MonthlyHistoricalNormals": {
+      "type": "object",
+      "properties": {
+        "temperature": {
+          "$ref": "#/definitions/WeatherUnitMaxMinAvg",
+          "description": "Temperature values."
+        },
+        "degreeDaySummary": {
+          "$ref": "#/definitions/DegreeDaySummary",
+          "description": "Summary of heating or cooling degree day information."
```

Comment:
Please change the description for degreeDaySummary to:

Summary of heating or cooling degree day information. Degree days are measures of how cold or warm a location is. A degree day compares the mean (the average of the high and low) outdoor temperatures recorded for a location to a standard temperature of 65 degrees F/ 18 degree C.